### ISSUE TITLE
`sortedSwitchCases` sometimes breaks return value

### ISSUE THREAD

**Issue by Semty** [nicklockwood/SwiftFormat#1453]
It happens sometimes, not always, but it's really annoying. 

Let's imagine that we have this enum case:
```swift
case .b, .a, .c, .e, .d:
    return nil
```
After calling `swiftformat .` with `sortedSwitchCases` enabled, sometimes it will generate this code:
```swift
case .a, .b, .c, .d, .e: return nil
```
Sometimes it will even generate this code:
```swift
case .a, .b, .c, .d,
    .e: return nil
```
And sometimes it will do the right job:
```swift
case .a, .b, .c, .d, .e: 
    return nil
```

**Comment by nicklockwood** [comment]
@facumenzella could you perhaps take a look at this? I can see why it's happening, but I'm not clear exactly what the original intention of the code was.

**Comment by facumenzella** [comment]
I'll take a look

**Comment by nicklockwood** [comment]
@facumenzella thanks! here's a test case:

```swift
    func testSortedSwitchCasesNoUnwrapReturn() {
        let input = """
        switch self {
        case .b, .a, .c, .e, .d:
            return nil
        }
        """
        let output = """
        switch self {
        case .a, .b, .c, .d, .e:
            return nil
        }
        """
        testFormatting(for: input, output, rule: FormatRules.sortedSwitchCases,
                       exclude: ["wrapSwitchCases"])
    }
```

**Comment by nicklockwood** [comment]
@Semty fixed in 0.51.10
